_header

```diff
@@ -0,0 +1,10 @@
+Cache-Control: no-cache, no-store, max-age=0, must-revalidate
+Pragma: no-cache
+Expires: 0
+X-Content-Type-Options: nosniff
+X-Frame-Options: DENY
```

Comment:
I don't see a reason to prevent this page from being included in an iFrame.

Reply:
This is a basic security measure. When designing for security, you should not ask for reasons to make something secure, but for reasons why it should be possible to frame this site. There might be legitimate reasons for doing so, but even then you want to control the sites being allowed to do so. In this case, X-Frame-Options: DENY is a defense against clickjacking (https://www.owasp.org/index.php/Clickjacking_Defense_Cheat_Sheet), addressing A1:2017- Injection in the OWASP Top 10 2017.

Reply:
As this is an open-source project (with Apache license) I believe people should be able to use it in any ways they please. And I'm totally ok with it being framed in another website.
As it is a static webpage, where you cannot trigger any dangerous actions, I believe this security measure is not needed. Please remove this header.

Reply:
As we found at least one possible attack vector. Click-jacking the download and providing malicious content, I'm convinced that even for a static website `X-Frame-Options: DENY` is the way to go. Thank you for keeping to it.